<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>false</bool>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Plotgrabber</string>
  </property>
  <property name="windowIcon">
   <iconset resource="myres.qrc">
    <normaloff>:/new/Images/Images/logo_Otic.png</normaloff>:/new/Images/Images/logo_Otic.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="6">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="11">
       <spacer name="horizontalSpacer_6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="9">
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="7">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="8">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="11">
       <widget class="QGraphicsView" name="graphicsView_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>256</height>
         </size>
        </property>
        <property name="verticalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="horizontalScrollBarPolicy">
         <enum>Qt::ScrollBarAlwaysOff</enum>
        </property>
        <property name="interactive">
         <bool>false</bool>
        </property>
        <property name="transformationAnchor">
         <enum>QGraphicsView::NoAnchor</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::BoundingRectViewportUpdate</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="5" rowspan="2" colspan="5">
       <widget class="MyView" name="graphicsView">
        <property name="mouseTracking">
         <bool>true</bool>
        </property>
        <property name="dragMode">
         <enum>QGraphicsView::RubberBandDrag</enum>
        </property>
        <property name="viewportUpdateMode">
         <enum>QGraphicsView::MinimalViewportUpdate</enum>
        </property>
        <property name="rubberBandSelectionMode">
         <enum>Qt::ContainsItemBoundingRect</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="11">
       <widget class="QTabWidget" name="tabWidget">
        <property name="maximumSize">
         <size>
          <width>256</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Data</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>0</number>
            </property>
            <item>
             <widget class="MyTable" name="tableWidget">
              <attribute name="verticalHeaderDefaultSectionSize">
               <number>20</number>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>XY</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <layout class="QVBoxLayout" name="verticalLayout_5">
            <item>
             <widget class="QGroupBox" name="groupBox">
              <property name="title">
               <string>X axis</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_X">
                 <property name="text">
                  <string>Log</string>
                 </property>
                </widget>
               </item>
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_3">
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="lineEdit"/>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_2"/>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label">
                   <property name="text">
                    <string>X1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_2">
                   <property name="text">
                    <string>X2</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="groupBox_2">
              <property name="title">
               <string>Y axis</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_7">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_6">
                 <item row="1" column="1">
                  <widget class="QLineEdit" name="lineEdit_4"/>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLineEdit" name="lineEdit_3"/>
                 </item>
                 <item row="0" column="0">
                  <widget class="QLabel" name="label_3">
                   <property name="text">
                    <string>Y1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="0" column="1">
                  <widget class="QLabel" name="label_4">
                   <property name="text">
                    <string>Y2</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <widget class="QCheckBox" name="checkBox_Y">
                 <property name="text">
                  <string>Log</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
          <item row="2" column="0">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="0" column="0">
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_3">
         <attribute name="title">
          <string>rθ</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_9">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout_8">
            <item row="3" column="0">
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QGroupBox" name="groupBox_3">
              <property name="title">
               <string>r</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="1" column="0">
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="2" column="0">
                  <widget class="QLineEdit" name="lineEdit_5"/>
                 </item>
                 <item row="1" column="1">
                  <widget class="QLabel" name="label_6">
                   <property name="text">
                    <string>r2</string>
                   </property>
                  </widget>
                 </item>
                 <item row="1" column="0">
                  <widget class="QLabel" name="label_5">
                   <property name="text">
                    <string>r1</string>
                   </property>
                  </widget>
                 </item>
                 <item row="2" column="1">
                  <widget class="QLineEdit" name="lineEdit_6"/>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0">
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="2" column="0">
             <widget class="QGroupBox" name="groupBox_4">
              <property name="title">
               <string>θ</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QRadioButton" name="radioButton">
                 <property name="text">
                  <string>Rad</string>
                 </property>
                 <property name="checked">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
               <item>
                <layout class="QHBoxLayout" name="horizontalLayout">
                 <item>
                  <widget class="QRadioButton" name="radioButton_2">
                   <property name="text">
                    <string>Deg</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_4">
         <attribute name="title">
          <string>Output</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QTextEdit" name="textEdit"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen_file"/>
    <addaction name="actionImage_from_clipboard"/>
    <addaction name="actionSave_data"/>
    <addaction name="actionClose_workspace"/>
    <addaction name="actionExit_program"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuImage">
    <property name="title">
     <string>Image</string>
    </property>
    <widget class="QMenu" name="menuRotate">
     <property name="title">
      <string>Rotate</string>
     </property>
     <addaction name="actionLeft"/>
     <addaction name="actionRight"/>
     <addaction name="actionUpside_down"/>
     <addaction name="actionArbitrary"/>
    </widget>
    <addaction name="menuRotate"/>
   </widget>
   <widget class="QMenu" name="menuPlot">
    <property name="title">
     <string>Plot</string>
    </property>
    <addaction name="actionSet_XY"/>
    <addaction name="actionSet_polar_plot"/>
    <addaction name="actionAnalyze"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuImage"/>
   <addaction name="menuPlot"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="acceptDrops">
    <bool>true</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_file"/>
   <addaction name="actionSave_data"/>
   <addaction name="actionSet_XY"/>
   <addaction name="actionSet_polar_plot"/>
   <addaction name="actionAnalyze"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen_file">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/actions/edit_add.png</normaloff>:/new/Images/Images/png/actions/edit_add.png</iconset>
   </property>
   <property name="text">
    <string>Open file</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/minetypes/info.png</normaloff>:/new/Images/Images/png/minetypes/info.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSet_XY">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/xy.png</normaloff>:/new/Images/Images/xy.png</iconset>
   </property>
   <property name="text">
    <string>Set cartesian plot</string>
   </property>
  </action>
  <action name="actionSelect_points">
   <property name="text">
    <string>Select points</string>
   </property>
  </action>
  <action name="actionAnalyze">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/actions/run.png</normaloff>:/new/Images/Images/png/actions/run.png</iconset>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </action>
  <action name="actionSave_data">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/apps/download_manager.png</normaloff>:/new/Images/Images/png/apps/download_manager.png</iconset>
   </property>
   <property name="text">
    <string>Save data</string>
   </property>
  </action>
  <action name="actionClose_workspace">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/actions/button_cancel.png</normaloff>:/new/Images/Images/png/actions/button_cancel.png</iconset>
   </property>
   <property name="text">
    <string>Close workspace</string>
   </property>
  </action>
  <action name="actionExit_program">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/actions/exit.png</normaloff>:/new/Images/Images/png/actions/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit program</string>
   </property>
  </action>
  <action name="actionSet_polar_plot">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/rphi.png</normaloff>:/new/Images/Images/rphi.png</iconset>
   </property>
   <property name="text">
    <string>Set polar plot</string>
   </property>
  </action>
  <action name="actionImage_from_clipboard">
   <property name="icon">
    <iconset resource="myres.qrc">
     <normaloff>:/new/Images/Images/png/actions/editcopy.png</normaloff>:/new/Images/Images/png/actions/editcopy.png</iconset>
   </property>
   <property name="text">
    <string>Image from clipboard</string>
   </property>
  </action>
  <action name="actionLeft">
   <property name="text">
    <string>Left</string>
   </property>
  </action>
  <action name="actionRight">
   <property name="text">
    <string>Right</string>
   </property>
  </action>
  <action name="actionUpside_down">
   <property name="text">
    <string>180°</string>
   </property>
  </action>
  <action name="actionArbitrary">
   <property name="text">
    <string>Arbitrary</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyView</class>
   <extends>QGraphicsView</extends>
   <header>myview.h</header>
  </customwidget>
  <customwidget>
   <class>MyTable</class>
   <extends>QTableWidget</extends>
   <header>mytable.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="myres.qrc"/>
 </resources>
 <connections/>
</ui>
